locales-ar.xml

```diff
@@ -199,75 +199,75 @@
 
     <!-- LONG ROLE FORMS -->
     <term name="director">
-      <single>director</single>
-      <multiple>directors</multiple>
+      <single>مدير</single>
+      <multiple>مديرون</multiple>
```

Comment:
@abdealikhurrum, can you confirm that you want:

```xml
    <term name="director">
      <single>مدير</single>
      <multiple>مديرون</multiple>
    </term>
```

instead of

```xml
    <term name="director">
      <single>مدور</single>
      <multiple>مدورين</multiple>
    </term>
```

Reply:
(it looks like you started editing from a rather old copy in which the director term hadn't been translated yet)

Reply:
Ah, it turns out that you contributed the second translation 2 years ago! :)

https://github.com/citation-style-language/locales/commit/a839348d7641d5a497da2102245b643d1a124c28